x-pack/plugins/lens/public/xy_visualization/reference_line_helpers.tsx

```diff
@@ -268,13 +311,18 @@ export const getReferenceSupportedLayer = (
       label: 'x' as const,
     },
   ];
+
+  const layers = state?.layers || [];
```

Comment:
nit: you can move it to line 323 to narrow the scope and use `layers` in line 339 here:
`        label: getAxisName(label, { isHorizontal: isHorizontalChart(state?.layers || []) }),`

Reply:
Thanks for this nit. It made the code cleaner 👍